### ISSUE TITLE
Lollypop can't index .wma music files because of missing plug-in

### ISSUE THREAD

**Issue by Eonfge** [flathub/org.gnome.Lollypop#91]
Found this one by accident:

```
[ERROR] 2020-07-28 18:43:41 Scanning file: file:///mnt/Data/Music/Band/album/number.wma, gst-core-error-quark: Your GStreamer installation is missing a plug-in. (12)
```

@bilelmoussaoui You know if we have to report this upstream, or if we must fix this here?

**Comment by bilelmoussaoui** [comment]
Upstream doesn't really define the gstreamer dependencies correctly on the meson.build file https://gitlab.gnome.org/World/lollypop/-/blob/master/meson.build#L22-28

It would be nice to open an issue regarding that upstream so we can know which gstreamer plugins we should ship here.

**Comment by Eonfge** [comment]
Notified Bellegarde. Should I also ask around at GNOME Build Meta, because apparently this codec is missing from the runtime while they have most other gst codecs?

**Comment by bellegarde-c** [comment]
I have no idea and no time to search. @Eonfge But I already give solution on GitLab in the past, you should be able to find.

**Comment by Eonfge** [comment]
Found it:
https://gitlab.gnome.org/World/lollypop/-/issues/1212

I'll see if I can research it further, but I'm really short on time so it might be next month before I can pick this up.

**Comment by Eonfge** [comment]
Looked into it. 

Previously (over a year ago), we still bundled versions of `gst-libav` and `gst-plugins-ugly` which made .wma files work. Now. If I reintroduce those versions (1.13.x) to the flatpak, then .wma files can be indexed and played. Now for the kick in the teeth, if I update those libs to the current version (1.16.2) then I can't play .wma files. Also checking the `gst-plugins-bad` module offers little help. It's just not there anymore :unamused: 

I'm not sure what happened to it. I'll make an issue about it upstream.

**Comment by Eonfge** [comment]
Fixed with #95
